app/views/admin/shared/_sidenav_users.html.haml

```diff
@@ -1,5 +1,5 @@
 %ul.sidebar-nav
-  - if current_ability.can?(:administer, User)
+  - if current_ability.can?(:manage_users, current_facility)
```

Comment:
I'm not sure if this is the correct check. If I'm senior staff or staff, right now, they see the well (`.sidebar-nav`), but it's empty.

A different issue, but once I click into a specific user, and go to the the "Payment Sources" tab, I see a "Add Payment Source" button, but clicking on it gives me a 403.